### PR TITLE
Measuring angles clockwise?

### PR DESCRIPTION
**Pull request by stgeorges**
Hi @chriswmackey , @mostaphaRoudsari ,

I was wondering if I have missed something in the last couple of months: was there a major change in the way Ladybug threats its cardinal direction angles? [Previously they were measured counter-clockwise](https://github.com/mostaphaRoudsari/ladybug/pull/110). Preparation.angle2north method still follows this convention. But I noticed that ResultVisualization.compassCircle method (which generates curves for the Ladybug "Sun Path" component) uses clockwise convention.

Also "hourlyCrvs: which Sunpath.drawSunPath (from Ladybug_ladybug.py) method generates and Ladybug "Sun Path" component uses, follow the same clock-wise convention (example: East = 1,0,0 and West = -1,0,0 instead of East = -1,0,0 and West = 1,0,0).

 Why does this discrepancy exist?

Thank you for the reply.

### CODE REVIEW

**Comment by mostaphaRoudsari** [comment]
Hi @stgeorges. Thanks you. It has been like this first release.

image

I agree that it's a good practice to have it all in the same order and I will merge this in, but is that going to effect the text? I don't remember how I was adding the text afterward.

Also you need to update the date of the component and send the userObject. Can you tell me again why can't you send the ghuser object with the pull request?

**Comment by stgeorges** [comment]
Hi @mostaphaRoudsari,
Change at line 2662: 
```
vector.Rotate(math.radians(angle), rc.Geometry.Vector3d.ZAxis)
```
will affect the text too (by text I mean "compassText" meshes).
I just wanted to be sure you will merge this commit, before attaching .ghuser file.
To my knowledge request can not be pulled (for .ghuser file or any other file) through github's web browser (which what I am using). You need to have either github windows application (which I can not install, because it supports only Windows XP> operating systems) or some other git client application (tried SmartGitHg, haven't succeeded). So I always make a commit, wait for your response, and then if you confirm that it will be merged, I attached the .ghuser file in comment like so:
http://www.filedropper.com/ladybugladybug

**Comment by chriswmackey** [comment]
@stgeorges ,
If you can confirm how and whether this change is affecting components like the sunpath (with the text placement), we can probably go ahead and merge it so that you can add the awesome PVWatts stuff.
-Chris

**Comment by stgeorges** [comment]
@chriswmackey , @mostaphaRoudsari 
The "-" change which this pull request requires will basically change the orientation of the "SunPath" components compass curves. The current state is on the left. The state that will appear after this pull request has been merged is the one on the left:

compass_curves_north_orient

What I didn't see up until now, is that sun positions per hour would have to be edited too. This means that changes in the "Sunpath" class from Ladybug_ladybug.py file will have to be made to. I am not willing to do this work.
So basically, I will just stick with the current sunPath compass curves orientation. Sorry for the inconvenience.

**Comment by mostaphaRoudsari** [comment]
Thank you @stgeorges for checking this. The PVWatts component made quite bit of an excitement and I'm looking forward to see it in action soon. Cheers- Mostapha the dictator! ;)

**Comment by mostaphaRoudsari** [comment]
Haha! https://www.youtube.com/watch?v=-uimGuWTX4M
